src/styles/less/Hud.less

```diff
@@ -1,272 +1,273 @@
+  }
+  &:nth-last-of-type(1) {
+    border-right: none;
+  }
 }
 
-.activity .checkImg{
-   display: none;
+.activity .checkImg {
+  display: none;
 }
 
 // Checkmark img for completed activity to be added after made in house
 .completedActivity {
-    // background-color: var(--green);
-    //border-radius: 1px 0px 0px 0px;
-    width: 16.666%;
-    text-align: center;
-    // This is how the I made the border angled on one side
-    transform: skewX(-15deg);
-		padding: 8px;
-   // border-right: 3px solid black;
-    // This is to make the element seem less italicized
-    span {
-        transform: skewX(15deg);
-    }
-    &:nth-last-of-type(1) {
-        border-right: none;
-    }
+  // background-color: var(--green);
+  //border-radius: 1px 0px 0px 0px;
+  width: 16.666%;
+  text-align: center;
+  // This is how the I made the border angled on one side
+  transform: skewX(-15deg);
+  padding: 8px;
+  // border-right: 3px solid black;
+  // This is to make the element seem less italicized
+  span {
+    transform: skewX(15deg);
+  }
+  &:nth-last-of-type(1) {
+    border-right: none;
+  }
 }
 
-.completedActivity .checkImg{
-    display: inline;
+.completedActivity .checkImg {
+  display: inline;
 }
 
-
-
 .currentActivity {
-    background-color:  var(--white);
-    width: 16.666%;
-    text-align: center;
-    // This is how the I made the border angled on one side
-    transform: skewX(-15deg);
-    border: 8px solid #C6FD7E;;
-    //border-right: 3px solid black;
-    // This is to make the element seem less italicized
-    span {
-        transform: skewX(15deg);
-    }
-    &:nth-last-of-type(1) {
-        border-right: none;
-    }
-    animation: pulse-animation 2s infinite;
-}
-
-.currentActivity .checkImg{
-    display: none;
+  background-color: var(--white);
+  width: 16.666%;
+  text-align: center;
+  // This is how the I made the border angled on one side
+  transform: skewX(-15deg);
+  border: 8px solid #c6fd7e;
+  //border-right: 3px solid black;
+  // This is to make the element seem less italicized
+  span {
+    transform: skewX(15deg);
+  }
+  &:nth-last-of-type(1) {
+    border-right: none;
+  }
+  animation: pulse-animation 2s infinite;
+}
+
+.currentActivity .checkImg {
+  display: none;
 }
 
 .restActive {
-	padding: 8px;
+  padding: 8px;
 }
 
 .restActive {
-	padding: 8px;
+  padding: 8px;
 }
 
 .restActive {
-	padding: 8px;
+  padding: 8px;
 }
 
 // This is the animation for current activity
 .pulse {
-    animation: pulse-animation 2s infinite;
+  animation: pulse-animation 2s infinite;
 }
 
 @keyframes pulse-animation {
-    0% {
-        box-shadow: 0 0 0 0px rgba(255, 255, 255, 0.4);
-    }
-    100% {
-        box-shadow: 0 0 0 20px rgba(255, 255, 255, 0);
-    }
+  0% {
+    box-shadow: 0 0 0 0px rgba(255, 255, 255, 0.4);
+  }
+  100% {
+    box-shadow: 0 0 0 20px rgba(255, 255, 255, 0);
+  }
 }
 
 .dayBars {
-	display: flex;
-	margin-top: 1%;
+  display: flex;
+  margin-top: 1%;
+
+  @media @mobileLarge {
```

Comment:
Please change this to @media screen and (max-width: 500px) to match the other media query in the file so both changes happen at the same time.

Reply:
Done.